docs/RELEASE-NOTES.rst

```diff
@@ -22,6 +22,7 @@ v1.1.0
 
 Added Functionality
 ```````````````````
+* Supports API changes introduced with Marathon v1.5.2
```

Comment:
Can you create a new v1.1.1 section in here with a Bug Fixes sub-section that calls out the specific bug and the github link to the issue? See http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/v1.1/RELEASE-NOTES.html for an example.

Reply:
Done!